app/src/main/res/layout/library_site_item.xml

```diff
@@ -15,7 +15,7 @@
             android:id="@+id/icon"
             android:layout_width="@dimen/history_favicon_width_height"
             android:layout_height="@dimen/history_favicon_width_height"
```

Comment:
I generally understand we are using 8dp here instead of 16dp because the width/height for the icon is bigger than 24x24. Admittedly, it is notable characteristic of how this xml was written, but I don't have any designs for this checkmark to see if we could just adjust all the icons here to 24x24 and remove the padding in the ImageView.

Reply:
We could do that by just manually changing the vector size. But to keep the 16 padding with would need to make `ImageSwitcher.width` of 24, which would make the checkmark icon smaller as well, almost twice as small.

Reply:
Right, I am not expecting any changes for this at this time.